saleor/payment/tests/test_utils.py

```diff
@@ -1353,3 +1353,92 @@ def test_create_transaction_event_for_transaction_session_call_webhook_for_fully
     flush_post_commit_hooks()
     mock_order_fully_paid.assert_called_once_with(order_with_lines)
     mock_order_updated.assert_called_once_with(order_with_lines)
+
+
+@pytest.mark.parametrize(
+    "response_result, transaction_amount_field_name",
+    [
+        (TransactionEventType.AUTHORIZATION_REQUEST, "authorize_pending_value"),
+        (TransactionEventType.AUTHORIZATION_SUCCESS, "authorized_value"),
+        (TransactionEventType.CHARGE_REQUEST, "charge_pending_value"),
+        (TransactionEventType.CHARGE_SUCCESS, "charged_value"),
+    ],
+)
+def test_create_transaction_event_for_transaction_session_success_sets_actions(
+    response_result,
+    transaction_amount_field_name,
```

Comment:
It looks like you are not using `transaction_amount_field_name` anywhere in this test

Reply:
Hhm, yeah you're right. thanks!